web/src/components/Cards/jobShowHeader.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 import  Card  from 'react-bootstrap/Card'
 
-const JobCardHeader = ({ category, title, styling }) => (
+const JobCardHeader = ({ category, role, styling }) => (
   <Card body className={ styling }>
-    { category }  >   { title }  
+    { category }  >   { role }  
```

Comment:
We're not likely to have the information available to be able to do this. We might get jobs listed as "Code Ninja". For now at least, we need to just take the listing titles at their word.